test/functional/feature_pruning.py

```diff
@@ -21,19 +24,60 @@
 # compatible with pruning based on key creation time.
 TIMESTAMP_WINDOW = 2 * 60 * 60
 
+def mine_large_blocks(node, n):
+    # Make a large scriptsig for the coinbase transaction. This is OP_RETURN
+    # followed by 950k of OP_NOP. This is a non-standard transaction but it
```

Comment:
one hand clapping: can a coinbase tx ever be "non-standard"? :)

Reply:
I think all coinbases are "non-standard", since we never relay them. :)

Reply:
Heh. Would you prefer "A transaction containing this scriptPubKey would be non-standard in a non-coinbase transaction, but is consensus valid"?

I also notice that the comment incorrectly says 'scriptsig' instead of 'scriptPubKey'. I'll update.

Reply:
just a joke :)

Reply:
Needed to fix other comments, so I've fixed this too. Thanks.